cfg/config_util.go

```diff
@@ -61,3 +61,16 @@ func IsMetricsEnabled(c *MetricsConfig) bool {
 func IsGKEEnvironment(mountPoint string) bool {
 	return strings.HasPrefix(mountPoint, "/dev/fd/")
 }
+
+// GetBucketType converts BucketType boolean flags to a BucketType enum.
+// The priority order is: Zonal > Hierarchical > Flat.
+// This is used to determine bucket-specific optimizations for kernel configs.
```

Comment:
we always wanted to convert the buckeTType struct into enum like this. Can you add a todo and take it after this PR to convert bucketType struct in buckeT_handle.go to this enum